src/worker/jobUpdate/updateJobID.py

```diff
@@ -0,0 +1,125 @@
+import os
+import sys
+import json
+import logging
+
+path = os.path.dirname(__file__)
+if path:
+    path = path + '/'
+p = os.path.abspath(path + '../helpers')
+sys.path.insert(1, p)
+from shmem_helper import Shared_Mem_Mngr
+from lock_helper import file_lock_mgr
+
+JobShm = {
+    "jobid": [],
+    "GPU": {},
+    "HCA": {}
+}
+
+# logging.basicConfig(level=logging.Info)
```

Comment:
should this line be uncommented?

Reply:
The line should be gone actually let me address